rc/base/auto-pairs.kak

```diff
@@ -0,0 +1,54 @@
+decl str-list auto_pairs '(,):{,}:[,]:<lt>,<gt>'
+
+def -hidden -params 2 auto-pairs-insert-opener %{
+  try %{
+    exec -draft "<a-K>[[:alnum:]]<ret>"
+    exec "%arg{1}%arg{2}<left>"
+  } catch %{
+    exec %arg{1}
+  }
+}
+
+def -hidden -params 2 auto-pairs-insert-closer %{
+  try %{
+    exec -draft "<a-k>\Q%arg{2}<ret>"
+    exec <right>
+  } catch %{
+    exec %arg{2}
+  }
+}
+
+def -hidden auto-pairs-insert-backspace %{
+  try %{
+    %sh{
+      regex='\Q'$(echo "$kak_opt_auto_pairs" | sed s/:/'\\E|\\Q'/g';'s/,//g)
+      echo "exec -draft 'H<a-k>$regex<ret>d'"
+    }
+  } catch %{
+    exec <backspace>
+  }
+}
+
+def auto-pairs-enable %{
+  %sh{
+    for pair in $(echo "$kak_opt_auto_pairs" | tr : '\n'); do
+      opener=$(echo $pair | cut --delimiter , --fields 1)
+      closer=$(echo $pair | cut --delimiter , --fields 2)
```

Comment:
Long options are not defined by [the standard](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/cut.html), we better use `-d` and `-f` here. Same remark for the following calls.